Needlework.Net.Desktop/ViewModels/ConsoleViewModel.cs

```diff
@@ -60,18 +60,25 @@ private async Task SendRequest()
                 var processInfo = Connector.GetProcessInfo();
                 var requestBody = WeakReferenceMessenger.Default.Send(new ContentRequestMessage(), "ConsoleRequestEditor").Response;
                 var content = new StringContent(Regex.Replace(requestBody, @"\s+", ""), new System.Net.Http.Headers.MediaTypeHeaderValue("application/json"));
-                var response = await Connector.SendAsync(method, RequestPath, content) ?? throw new Exception("Response is null.");
+                var response = await Connector.SendAsync(method, RequestPath, content);
                 var riotAuthentication = new RiotAuthentication(processInfo.RemotingAuthToken);
-                var body = await response.Content.ReadAsStringAsync();
-
-                body = !string.IsNullOrEmpty(body) ? JsonSerializer.Serialize(JsonSerializer.Deserialize<object>(body), App.JsonSerializerOptions) : string.Empty;
+                var responseBody =
+#if DEBUG 
+                    await response.Content.ReadAsStringAsync();
+#else
+                    await response.Content.ReadAsByteArrayAsync();
+#endif
```

Comment:
What's the purpose behind this? I'm not familiar with preprocessor directives.

Reply:
I thought you used the `ReadAsStringAsync` method on purpose (maybe to look at the original value ... or something like that), so I decided to leave it there in debug mode, and use `ReadAsByteArrayAsync` in release mode for better performance (`ReadAsByteArrayAsync` is better because `JsonSerializer` works with utf8 strings, so you don't have to unnecessarily turn `byte[]` into `string` and then back `string` into `byte[]`)

if you don't plan to check what value is returned from `ReadAsStringAsync` then I will just leave `ReadAsByteArrayAsync` there